### DISCUSSION TITLE
Redirect unknown Host works on http but not on https

### DISCUSSION THREAD

**Discussion by jpdsc** [NginxProxyManager/nginx-proxy-manager#1241] [Q&A]
Hello there,

I have set the redirect option to a placeholder subdomain to ensure unknown Hosts are redirected.
When going to the subdomain in http it gets re-directed correctly. However, when going to the subdomain in https I get a certificate warning instead of a redirect.

http://example1.domain.com > https://redirected.domain.com 
https://example1.domain.com > certificate warning

https://redirected.domain.com has a SSL certificate and the HTTPS options enabled

Am I doing something wrong?

Thank you.

Joao

EDIT: found the answer here: https://github.com/jc21/nginx-proxy-manager/issues/657

**Comment by jc21** [comment]
The https protocol MUST have a valid certificate matching the domain requested from the browser. You can't just have a catch all ssl for any unknown incoming domain or address.

What you can do is create a redirect for all subdomains of the top level domain, *.domain.com and create a ssl cert for that to use.

If any server was able to create valid signed ssl certificates for any domain at all, then https/ssl would t be very secure anymore.
